### ISSUE TITLE
Demo on Anthos GKE for AWS

### ISSUE THREAD

**Issue by PsychoSid** [GoogleCloudPlatform/bank-of-anthos#376]
Hi,

I know the repo is GCP based but before v0.3 it worked fine on a GKE on AWS cluster. Now it fails to start many services have:-

```
2020-09-30 08:24:40 | [ERROR] | report |

***************************
APPLICATION FAILED TO START
***************************

Description:

Invalid Micrometer configuration detected:

  - stackdriver.projectId was 'null' but it is required

Action:

Update your application to correct the invalid configuration.
```

I guess this is expected ? Is there an alternative for GKE on AWS ?

Thanks,

**Comment by askmeegs** [comment]
Hi @PsychoSid, sorry you're seeing this breaking change - could you try to configure `ENABLE_METRICS=false` in the Kubernetes manifests for `ledgerwriter`, `balancereader`, and `transactionhistory` ?  (By default, that field is set to true, which means the Java services are trying to authenticate to the Cloud Monitoring API to send app-level Micrometer metrics to GCP.) 

Not sure if that will fix it, we may have to update the Micrometer code.

**Comment by PsychoSid** [comment]
Hi,

Sorry it's taken a while to come back to you. I set that variable to "false" the issue remains unfortunately.

Thanks.

**Comment by askmeegs** [comment]
Hi @PsychoSid - I think I resolved this startup crash. I've tested the fix on my EKS cluster.

Continue setting `ENABLE_TRACING=false` and `ENABLE_METRICS=false`, and also replace your images, for now, to use the `latest` tag (instead of `v0.3.0`.) 

This patch will be included in an upcoming `v0.3.x` release.

**Comment by gfthybridlabs** [comment]
Hi @askmeegs can confirm the workaround above works for me on an AWS Anthos GKE cluster. Thanks.
